hardware/Wireless-Products-Protocols-and-Applications.md

```diff
@@ -1,54 +1,61 @@
 - Chip name conventions:
 
   For proprietary products, there are many FG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
-
-  ![](files/HW-Wireless-Products-Protocols-and-Applications/NameConvention-1.png)
-
-
+  <div align="center">
+  <img src="files\HW-Wireless-Products-Protocols-and-Applications\NameConvention-1.png">  
+  </div>
+  
   As you can see from the name convention, the proprietary chips include four series such as FG1,FG12,FG13,and FG14. The chips are classified with  P (Performance), B (Basic), and V (Value) performance grades. The chips are Tx/Rx type which include sub-G band, 2.4GHz band, and sub-G& 2.4GHz bands. The chips are classified with 2 temperature range as -40°-+85° and -40°-+125°. And there are many packages(QFN32, QFN48, QFN68, and BGA125) to select. 
 - Applications:
 
-  Meters(Gas, water, electricity), Door lockers, Electronic lables, Wireless toy, Wireless ranch, Remote controllers, etc.
-## Zigbee/Thread products and applications
+  Meters(Gas, water, electricity), Door locks, Electronic lables, Wireless toys, Wireless ranch, Remote controllers, etc.
+# 2. Zigbee/Thread products and applications
 - Introduction: 
 
-  Silicon Labs is the vendor of choice for OEMs developing Zigbee networking into their products. The Silicon Labs Zigbee platform is the most integrated, complete and feature rich Zigbee solution available — a family of Wireless SoCs, based on Arm® Cortex® processor and 2.4 GHz transceiver, together the most reliable, scalable and advanced Zigbee software and supported by best-in-class development tools.
+  Silicon Labs is the vendor of choice for OEMs developing Zigbee networking into their products. The Silicon Labs Zigbee platform is the most integrated, complete and feature rich Zigbee solution available — a family of Wireless SoCs, based on Arm® Cortex® processor and 2.4 GHz transceiver, together with the most reliable, scalable and advanced Zigbee software and supported by best-in-class development tools.
 - Chip name conventions:
 
   For Zigbee/Thread products, there are also many MG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
-![](files/HW-Wireless-Products-Protocols-and-Applications/NameConvention-2.png)
+  <div align="center">
+  <img src="files\HW-Wireless-Products-Protocols-and-Applications\NameConvention-2.png">  
+  </div>
+
   As you can see from the name convention, the Zigbee/Thread chips include 2 series such as MG12 and MG13. The Zigbee/Thread chips only have  P performance grade. The chips are Tx/Rx type which include sub-G band, 2.4GHz band, and sub-G& 2.4GHz bands. The chips are classified with 2 temperature range as -40°-+85° and -40°-+125°. And there are many packages(QFN32, QFN48, QFN68, and BGA125) to select. 
```

Comment:
"2 series such as MG12 and MG13" is not accurate.  We should use series to distinguish series 1 EFR32MG1x and series 2 EFR32MG2x.  MG12 and MG13 are different variants.  Do we want to mention series 2 here as well?  If so, we should include QFN40 in the package list.

Reply:
Modified.